.github/workflows/www.yaml

```diff
@@ -14,6 +14,28 @@ on:
       - ".github/workflows/www.yaml"
       - "www/**"
 jobs:
+  unit:
+    name: "Unit Tests"
```

Comment:
If we are using name here, can you add one for lint too?

Reply:
Done